src/SoapCore/Meta/MetaBodyWriter.cs

```diff
@@ -875,7 +876,20 @@ private void AddSchemaTypePropertyOrField(XmlDictionaryWriter writer, MemberInfo
 			}
 			else
 			{
-				AddSchemaType(writer, toBuild, parentTypeToBuild.ChildElementName ?? member.Name, isArray: createListWithoutProxyType, isListWithoutWrapper: createListWithoutProxyType, isUnqualified: isUnqualified);
+				string defaultValue = null;
+				var defaultAttribute = member.GetCustomAttribute<DefaultValueAttribute>();
+				if (defaultAttribute != null)
+				{
+					if (defaultAttribute.Value is bool value)
+					{
+						defaultValue = value ? "true" : "false";
+					}
+					else
+					{
+						defaultValue = defaultAttribute.Value.ToString();

```

Comment:
```suggestion
						defaultValue = defaultAttribute.Value?.ToString();

```
Value can be null

Reply:
Good spot :).

I've made a change to match the behaviour in .NET framework ASMX services. It looks like [DefaultValue(null)] is equivalent to not supplying the attribute.